src/Repository/Repository.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Repository;
+
+use App\Repository\DBConnexion;
+
+abstract class Repository
+{
+    protected DBConnexion $dbConnection;
+    protected $table;
+
+    public function __construct(DBConnexion $dbConnection)
```

Comment:
table should be pass to the construct to have it has mandatory information for any repository.